static/js/app.ts

```diff
@@ -1590,11 +1617,12 @@ export function toggle_developers_mode(enforced: boolean) {
     $('#editor-area').removeClass('mt-5');
     $('#code_editor').css('height', 36 + "em");
     $('#code_output').css('height', 36 + "em");
-    theGlobalEditor.resize();
+    theGlobalEditor.resize(576);
```

Comment:
Fixed pixel sizes are a bit ewww. Is this necessary?

Reply:
Yes, don't love them. But resizing this editor is quite difficult, it doesn't quite interact with the parent in the same way that Ace does. To be able to get rid of this, I think in the future we can implement the errors as CodeMirror widgets.